src/lib/Component/TwigComponent.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\TwigComponents\Component;
+
+use Ibexa\Contracts\TwigComponents\ComponentInterface;
+use Twig\Environment;
+
+class TwigComponent implements ComponentInterface
```

Comment:
```suggestion
final class TwigComponent implements ComponentInterface
```

Reply:
All components are marked as lazy so I think is should be without 'final'